src/main/kotlin/com/example/travelermileageservice/application/ReviewService.kt

```diff
@@ -0,0 +1,21 @@
+package com.example.travelermileageservice.application
+
+import com.example.travelermileageservice.domain.Review
+import com.example.travelermileageservice.domain.repository.ReviewRepository
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import java.util.*
+
+@Service
+class ReviewService(
+    private val reviewRepository: ReviewRepository
+) {
+    @Transactional
+    fun save(review: Review) {
+        reviewRepository.save(review)
+    }
+
+    fun checkExistReview(reviewId: UUID): Boolean {
```

Comment:
`exist` 만으로 충분한 의미 전달이 될듯 합니다. 중의적 명명은 지양합시다.

```suggestion
    fun existReview(reviewId: UUID): Boolean {
```